qmxgraph/page/api.js

```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
     return model.isVisible(cell);
 };
 
+/**
+ * Sets graph connectable property.
+ *
+ * @param {number} cellId Id of a cell in graph.
+ * @param {boolean} enabled Enable value of this property.
+ */
+graphs.Api.prototype.setConnectable = function setConnectable(cellId, enabled) {
+    "use strict";
+
+    var graph = this._graphEditor.graph;
+    var model = graph.getModel();
+    var cell = this._findCell(model, cellId);
+    if (cell) {
+        cell.setConnectable(enabled);
+    } else {
+        throw Error("Unable to find cell with id " + cellId);
+    }
+};
+
+/**
+ * Returns current graph connectable property.
+ *
+ * @returns {boolean} Enable value of this property.
+ */
+graphs.Api.prototype.isConnectable = function isCellsConnectable(cellId) {
```

Comment:
This one too should be `isConnectable` I believe.

Reply:
🤦‍♂
Why didn't I check that also?

Reply:
I could have commented in both too.